src/GraphQL.ApiTests/ApiApprovalTests.PublicApi.GraphQL.approved.txt

```diff
@@ -338,8 +338,9 @@ namespace GraphQL
         public static object ConvertTo(object value, System.Type targetType) { }
         public static T ConvertTo<T>(object value) { }
         public static void Register(System.Type valueType, System.Type targetType, System.Func<object, object> conversion) { }
-        public static bool TryConvertTo(object value, System.Type targetType, out object result) { }
-        public static bool TryConvertTo<T>(object value, out T result) { }
```

Comment:
No need for these APIs to be public (yet). Not big problem to remove because these APIs were introduced in previous commit.